shared/cfg/base.cfg

```diff
@@ -313,13 +313,15 @@ image_unbootable_pattern = "Hard Disk.*not a bootable disk"
 # used to communicate with hypervisor to control guests.
 # Order cooresponds to 'vms' list above.
 monitors = qmpmonitor1
+chardev_backend_qmpmonitor1 = unix_socket
```

Comment:
How abou rename param  'chardev_backend_qmpmonitor1' to  'backend_type_qmpmonitor1'

Reply:
@xutian  It is a param that used to generate chardev, start with "chardev", then the chardev option, refer: https://github.com/avocado-framework/avocado-vt/blob/b4502d1d3e377187441ded79d53a995841f12296/virttest/qemu_devices/qcontainer.py#L2266
Many cases use this param, we can not change it.

Reply:
> How abou rename param 'chardev_backend_qmpmonitor1' to 'backend_type_qmpmonitor1'

like yfu said,  seems `chardev_backend_ ` is fixed format.